pkg/rtc/types/interfaces.go

```diff
@@ -348,23 +347,25 @@ type MediaTrack interface {
 	// returns quality information that's appropriate for width & height
 	GetQualityForDimension(width, height uint32) livekit.VideoQuality
 
-	NotifySubscriberNodeMaxQuality(nodeID livekit.NodeID, qualites []SubscribedCodecQuality)
-	NotifySubscriberNodeMediaLoss(nodeID livekit.NodeID, fractionalLoss uint8)
```

Comment:
These are moving from `MediaTrack` -> `LocalMediaTrack` as once again relay track will directly go to the relay maanger.